src/data/mongod.service

```diff
@@ -0,0 +1,33 @@
+[Unit]
+Description=MongoDB Database Server
+Documentation=https://docs.mongodb.org/manual
+After=network-online.target
+Wants=network-online.target
+
+[Service]
+User=mongodb
+Group=mongodb
+EnvironmentFile=-/etc/default/mongod
+ExecStart=/usr/bin/mongod --config /etc/mongod.conf
```

Comment:
You could replace this with the following, read this file the way Marc suggests above, and then format this line as a string: 
```ini
ExecStart=%s
```

This is overkill, but you could also use the [`ConfigParser`](https://docs.python.org/3/library/configparser.html) library to read this file into a dict-like object and modify it the same way you normally do with a dict.

Reply:
I am actually going to opt to read from `/lib/systemd/system/mongod.service` on the machine, modify it, then save it the modified version in `/etc/systemd/system/mongod.service`. So this file will no longer exist